pkg/api/customization/node/node.go

```diff
@@ -48,16 +48,35 @@ func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 	if nodeTemplateID == nil && customConfig == nil {
 		delete(resource.Links, "remove")
 	}
-	if resource.Values[client.NodeFieldWorker] != true ||
-		resource.Values["state"] == "registering" ||
-		resource.Values["state"] == "unavailable" {
+	if resource.Values[client.NodeFieldWorker] != true {
 		return
 	}
-	if resource.Values["state"] == "draining" {
+	state := resource.Values["state"]
+	allowed := state == "active" || state == "cordoned" || state == "draining" || state == "drained"
+	if !allowed {
+		return
+	}
+	if state == "draining" {
+		conditions := convert.ToMapSlice(resource.Values["conditions"])
```

Comment:
I believe there is a way to retrieve the condition using the norman condition helper. Can you check on that?

Reply:
not sure I can make use of it when fetching from `resource.Values` since I don't have node object here